django/db/backends/schema.py

```diff
@@ -268,6 +268,9 @@ def create_model(self, model):
         for fields in model._meta.unique_together:
             columns = [model._meta.get_field(field).column for field in fields]
             column_sqls.append(self.sql_create_table_unique % {
+                # ticket #24102: constraint name may be useful for some backends
```

Comment:
@timgraham -  cannot add a regression test right now as `django.db.backends.sqlite3.introspection.get_constraints` does not return constraint names.